constellations/orion/src/domains/user.go

```diff
@@ -53,5 +57,14 @@ func (user *User) Validate() error {
 		return errors.New("invalid phone")
 	}
 
+	// School validation
+	if matches, _ := regexp.MatchString(`[^a-zA-Z\s]`, school); matches && user.School.Valid {
```

Comment:
the `[a-zA-Z]` gets used often in our codebase that we actually have a constant for it. Can you please replace this part with one of the patterns in domains/regex.go?